packages/web/src/components/remove/remove-liquidity-select-list-item/RemoveLiquiditySelectListItem.tsx

```diff
@@ -26,26 +27,29 @@ interface TooltipProps {
   disabled: boolean;
 }
 
+const TokenValueWrapper = styled.div``;
+const TokenTitleWrapper = styled.div``;
```

Comment:
Can you move it to styles.ts in the same folder?
And please put the flex style in there, so we can be predictable.

Reply:
oh my bad, i will do it right now